core/src/tasks/profile/enqueueExports.ts

```diff
@@ -0,0 +1,49 @@
+import { Op } from "sequelize";
+import { Profile } from "../../models/Profile";
+import { CLSTask } from "../../classes/tasks/clsTask";
+import { plugin } from "../../modules/plugin";
+import { Import } from "../../models/Import";
+import { CLS } from "../../modules/cls";
+
+export class ProfilesEnqueueExports extends CLSTask {
+  constructor() {
+    super();
+    this.name = "profiles:enqueueExports";
+    this.description =
+      "If a profile is ready and has an import that has not exported, enqueue profile:export for them";
+    this.frequency = 1000 * 10;
+    this.queue = "profiles";
```

Comment:
👍 
The `profiles` queue is lower in priority than the `exports` and `exports:{destination}` queues in priority.  So, this task should run /after/ we've started to work on the exports, meaning the set of remaining profiles to export should change before this runs again.